_data/simple-icons.json

```
@@ -1275,6 +1275,11 @@
             "hex": "00C775",
             "source": "https://greenkeeper.io/"
         },
+        {
+            "title": "GreenSock",
+            "hex": "88CE02",
+            "source": "https://greensock.com/wp-content/themes/greensock/images/greensock-logo.svg"
```

I'd suggest changing this to their homepage as it contains the SVG. This is more likely to stay true then the URL you specified is likely to stay accurate.

```suggestion
            "source": "https://greensock.com/"
```

---

Updated the source per suggestion.